_posts/2021-08-25-debezium-1-7-beta1-released.adoc

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title:  Debezium 1.7.0.Beta1 Released
+date:   2021-08-25
+tags: [ releases, mysql, postgres, sqlserver, cassandra, oracle, db2, vitess, outbox, kafka ]
+author: rkerner
+---
+
+It's my pleasure to announce the first release of the Debezium 1.7 series, *1.7.0.Beta1*!
+
+This release brings https://docs.nats.io/developing-with-nats-streaming/streaming#nats-streaming-overview[NATS Streaming]
+support for Debezium Server along with many other fixes and enhancements. +
+Also this release is the first one tested with Apache Kafka 2.8.
+
++++<!-- more -->+++
+
+Also this release has a breaking change for the MySQL Connector. +
+The MySQL driver was updated to the latest version 8.0.26 with https://issues.jboss.org/browse/DBZ-3833[DBZ-3833].
+This update comes with a new timezone handling and configuration options. +
```

Comment:
How is the driver update breaking? New options doesn't sound harmful?

Reply:
according to flags in the Jira and according to @jpechane it is. I think properties for timezone handling changed?

Reply:
Ok, add a link to the upgrade procedure for that release (https://debezium.io/releases/1.7/release-notes#release-1.7.0-beta1), which should discuss this (and any other) breaking change.

Reply:
This paragraph has already the same information like that section in the release announcement. The change only affects people that manually provide DSN options or that use the MySQL connector as dependency. All the related information can be taken from the linked DBZ issue and the linked MySQL docs.

Nothing to do for average users that's not mentioned here. Can I leave it how it is?

Reply:
Yes, that's ok.